--- src/model/api.ts	2017-08-23 12:33:37.902200677 +0200
+++ src/model/api.ts	2017-08-23 12:30:56.243290118 +0200
@@ -43,7 +43,8 @@
 }
 
 
-export class BLECharacteristics extends Map<String, BLECharacteristic> {
+export class BLECharacteristics {
+    [key: string]: BLECharacteristic;
 }
 
 
@@ -77,7 +78,8 @@
 }
 
 
-export class BLEDescriptors extends Map<String, BLEDescriptor> {
+export class BLEDescriptors {
+    [key: string]: BLEDescriptor;
 }
 
 
@@ -142,7 +144,8 @@
 }
 
 
-export class BLEServices extends Map<String, BLEService> {
+export class BLEServices {
+    [key: string]: BLEService;
 }
 
 
